tools/perf-test-processing/src/index.ts

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fs from "fs";
+import path from "path";
+import { ITelemetryBufferedLogger } from "@fluidframework/test-driver-definitions";
+import { ConsoleLogger } from "./logger";
+
+// Allow for dynamic injection of a logger. Leveraged in internal CI pipelines.
+// The parameter to getTestLogger() is a delay to apply after flushing the buffer.
+const _global: any = global;
+let logger: ITelemetryBufferedLogger = _global.getTestLogger?.(5_000);
+
+if (logger === undefined) {
+    logger = new ConsoleLogger();
+}
+
+const filesToProcess: string[] = [];
+
+const dirs = process.argv.slice(2);
```

Comment:
can you add comments around how this works, and what the argument are in general

Reply:
i think our other tools use some library for processing cli arguments. its not the best, but i think it is better than diy, and keeping consistency between our tools is good

Reply:
Agreed in general. I did try to use OCLIF but this was quicker for the initial experiment. Thoughts on creating a separate item to move this tool to OCLIF? It will still need work for the memory tests anyway.

In the mean time I can add the comments around the use of `argv`.

Reply:
yeah. please file a pr, but this is fine for now

Reply:
I'm actually doing it as part of the PR to process the output of memory tests (https://github.com/microsoft/FluidFramework/pull/11665). Not pushed yet, but it'll be there.